src/librustc_bitflags/lib.rs

```diff
@@ -23,6 +24,7 @@
 
 //! A typesafe bitmask flag generator.
 
+extern crate core;
```

Comment:
My change makes this required everywhere.

Actually looking on the profiler again maybe I could do this in a better way.